test/either_test.dart

```diff
@@ -66,4 +66,18 @@ void main() {
     expect(right(1).toIterable().toList(), [1]);
     expect(left("nope").toIterable().toList(), []);
   });
+
+  group("Left", () {
+    test("value", () {
+      final some = Left(2);
```

Comment:
If these used either the convenience constructor `left` or `new Left(2)` then I think it would be Dart 1 compatible as well.